pkgs/applications/networking/cluster/kubernetes/dns.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub, go }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "kube-dns-${version}";
+  version = "1.14.1";
+
+  src = fetchFromGitHub {
+    owner = "kubernetes";
+    repo = "dns";
+    rev = "${version}";
+    sha256 = "13l42wm2rcz1ina8hhhagbnck6f1gdbwj33dmnrr52pwi1xh52f7";
+  };
+
+  buildInputs = [ go ];
+
+  buildPhase = ''
+    export GOPATH="$(pwd)/.gopath"
+    mkdir $GOPATH
+    ln -s $(pwd)/vendor $GOPATH/src
+    mkdir $GOPATH/src/k8s.io/dns
+    ln -s $(pwd)/cmd $GOPATH/src/k8s.io/dns/cmd
+    ln -s $(pwd)/pkg $GOPATH/src/k8s.io/dns/pkg
+    go build -o kube-dns ./cmd/kube-dns
```

Comment:
Ah okay, in that case I'd just leave a comment saying that so the next reader doesn't start from 0 if they want to try to convert it to use the standard stuff. I know the go builder does have the option to build only a subset of the targets. cc @kamilchm